ext/json/json_encoder.c

```diff
@@ -312,7 +308,7 @@ static int php_json_encode_array(smart_str *buf, zval *val, int options, php_jso
 }
 /* }}} */
 
-static int php_json_escape_string(
+int php_json_escape_string(
```

Comment:
The return type seems like it could be:
```suggestion
zend_result php_json_escape_string(
```

But then a lot of the JSON API seems to also want to be a ```zend_result`` so maybe that's best left for master